### PR TITLE
bug fix, indexing empty slice

### PR DESCRIPTION
**Pull request by MohitKS5**
Signed-off-by: MohitKS5 <mohitkumarsingh907@gmail.com>

### CODE REVIEW

**Comment by richvdh** [comment]
> Signed-off-by: MohitKS5

As per https://github.com/matrix-org/matrix-doc/blob/master/CONTRIBUTING.rst#sign-off, please use your real name here.

File: src/github.com/matrix-org/dendrite/roomserver/storage/event_state_keys_table.go [hunk 1]
```diff
@@ -130,7 +130,7 @@ func (s *eventStateKeyStatements) bulkSelectEventStateKeyNID(
 func (s *eventStateKeyStatements) bulkSelectEventStateKey(
 	ctx context.Context, eventStateKeyNIDs []types.EventStateKeyNID,
 ) (map[types.EventStateKeyNID]string, error) {
-	var nIDs pq.Int64Array
+	nIDs := make(pq.Int64Array, len(eventStateKeyNIDs))
```
[the remark this answers is not in the document]
Comment (MohitKS5) [reply src/github.com/matrix-org/dendrite/roomserver/storage/event_state_keys_table.go]: interpreted from line 135

[the remark this answers is not in the document]
Comment (richvdh) [reply src/github.com/matrix-org/dendrite/roomserver/storage/event_state_keys_table.go]: > interpreted from line 135

huh?

[the remark this answers is not in the document]
Comment (MohitKS5) [reply src/github.com/matrix-org/dendrite/roomserver/storage/event_state_keys_table.go]: may be I didn't get what you are saying ... 
nIDs is int64 array right

[the remark this answers is not in the document]
Comment (richvdh) [reply src/github.com/matrix-org/dendrite/roomserver/storage/event_state_keys_table.go]: nIDs is an `Int64Array`

so why are you making a `[]Int64Array`

[the remark this answers is not in the document]
Comment (richvdh) [reply src/github.com/matrix-org/dendrite/roomserver/storage/event_state_keys_table.go]: ohhh
I'm being stupid, sorry

**Comment by richvdh** [comment]
ok this looks correct. Sorry for being stupid.

Presumably this isn't working at all right now? What are the symptoms you are fixing?

Please sign it off.

**Comment by MohitKS5** [comment]
signed off with real name ....

**Comment by MohitKS5** [comment]
@anoadragon453  fixed this eleven days ago I guess.

**Comment by anoadragon453** [comment]
Oh whoops. Appreciate you pointing it out though!

**Pull request closed without merging**